charts/civil-sdt/values.preview.template.yaml

```diff
@@ -15,7 +15,14 @@ java:
     civil:
       resourceGroup: civil
       secrets:
-        - name: AppInsightsInstrumentationKey
+        - name: civil-sdt-service-s2s-secret
+          alias: CIVIL_SDT_S2S_KEY
+        - name: civil-sdt-POSTGRES-USER
+          alias: SDT_DB_USERNAME
+        - name: civil-sdt-POSTGRES-PASS
+          alias: SDT_DB_PASSWORD
+        - name: civil-sdt-servicebus-connection-string
+          alias: CIVIL_SDT_SERVICEBUS_CONNECTION_STRING
```

Comment:
Are these needed?  They are being set to the same as in values.yaml, so should be picked up from there.

Reply:
this is just a template file,